README.md

```diff
@@ -44,6 +44,14 @@ docker build -t xmpp-providers -f ./Dockerfile .
 docker run -p 80:80 -t -i xmpp-providers
 ```
 
+## Update Process
+
+There are several steps needed in order to see changes of the [provider data](https://invent.kde.org/melvo/xmpp-providers) on the [website](https://providers.xmpp.net):
```

Comment:
```suggestion
Changes of the [provider data](https://invent.kde.org/melvo/xmpp-providers) may take a little while to become visible on the [website](https://providers.xmpp.net), because there is some delay in building and deploying:
```

Reply:
```suggestion
Changes of the [provider data](https://invent.kde.org/melvo/xmpp-providers) may take a while to become visible on the [website](https://providers.xmpp.net).
This is due to some delay in building and deploying.
```